Contest_API.md

```diff
@@ -546,9 +546,9 @@ The following endpoint is associated with API version:
 
 | Endpoint | Mime-type        | Required? | Description
 | :------- | :--------------- | :-------- | :----------
-| `/`      | application/json | yes       | JSON object with version information as defined in the table below.
+| `/`      | application/json | yes       | JSON object with properties as defined in the table below.
```

Comment:
All other endpoint descriptions are on the form "JSON object of X with properties as defined in the table below". The X here should be "version information".

That said, the "JSON object of X" would sound a bit strange here, but it also sounds a bit strange everywhere, doesn't it?

Reply:
What do you think about "JSON object representing a single X"?

Reply:
Sorry, I see you gave a thumbs up in our slack :)